editors/sc-ide/core/sc_server.cpp

```diff
@@ -378,8 +393,12 @@ void ScServer::onScLangReponse( const QString & selector, const QString & data )
     static QString unmutedSelector("serverUnmuted");
     static QString ampSelector("serverAmp");
     static QString ampRangeSelector("serverAmpRange");
-	static QString startDumpOSCSelector("dumpOSCStarted");
-	static QString stopDumpOSCSelector("dumpOSCStopped");
+    static QString startDumpOSCSelector("dumpOSCStarted");
+    static QString stopDumpOSCSelector("dumpOSCStopped");
+    static QString startRecordingSelector("recordingStarted");
+    static QString pauseRecordingSelector("recordingPaused");
+    static QString stopRecordingSelector("recordingStopped");
+    static QString recordingDurationSelector("recordingDuration");
```

Comment:
http://woboq.com/blog/qstringliteral.html
http://doc.qt.io/qt-5/qstring.html#QStringLiteral
(google hits one and two)

`QString xxx = QStringLiteral("yyy")`